theano/configdefaults.py

```diff
@@ -1241,7 +1241,8 @@ def is_valid_check_preallocated_output_param(param):
 def check_mkl_openmp():
     if not theano.config.blas.check_openmp:
         return
-    import os
+    if sys.platform == 'darwin':
```

Comment:
Could we also check if this is a conda python or not? I don't think we have this logic

Reply:
We don't need that.

clang doesn't support OpenMP so even if you're not in conda you still won't have problems.

Or did you have something else in mind?